BreanaMarie/tic-tac-toe/ticTacToe.js

```diff
@@ -0,0 +1,458 @@
+		    	console.log("Space Jammer just went");
+		    	if(selectedSpace.hasClass('freeSpace')) { 
+			        selectedSpace.removeClass('freeSpace');
+			        selectedSpace.addClass('playedSpaceO');
+			        
+			        winSpaces();
+			        console.log(i);
+			        i ++;
+			        turn();
+		    	} 
+		    }
+		}
+		//if already playedSpace*
+		else if (selectedSpace.hasClass('playedSpaceX' || 'playedSpaceO')){
+			alert("this spot has already been played");
+			changeSpace();
+		}	
+	});
+}
+changeSpace();
+
+//create a reset button
+var resetSpaces =function(){
+	console.log("board has been reset");
+	$('div[name="space"]').removeClass('playedSpaceX');
+	$('div[name="space"]').removeClass('playedSpaceO');
+	$('div[name="space"]').removeClass('winnerSpace');
+	$('div[name="space"]').addClass('freeSpace');
+	round='WDI 23';
+	i=0;
+	$("p").html("The Game Begins with WDI 23");
+};
+
+//listen for reset button to be clicked
+$(".buttonReset").on('click', function(){
+	resetSpaces();
+});
+
+
+
+//returns which way was used to win
+var winSpaces = function(){
+	return (acrossTop() || acrossMid() || acrossBot()|| diagLeft() || diagRight() || downFirst() || downMid() || downLast());
+};
+
+//check three in a row change background color to show win
+//list ways to win
+//declare winner and reset
+var acrossTop= function($box1, $box2, $box3){
+	var space1 = $('#box1').attr("class");
+	var space2 = $('#box2').attr("class");
+	var space3 = $('#box3').attr("class");
+
+	if ((space1 === space2) && (space2 === space3)){
+		if (space1 === 'playedSpaceX'){
+			console.log ('X has won');
+			$('#box1').addClass('winnerSpace');
+			$('#box2').addClass('winnerSpace');
+			$('#box3').addClass('winnerSpace');
+			alert(round + " Has Won");
+			resetSpaces();
+			return acrossTop;
+		}
+		else if (space1 === 'playedSpaceO'){
+			console.log ('O has won');
+			$('#box1').addClass('winnerSpace');
+			$('#box2').addClass('winnerSpace');
+			$('#box3').addClass('winnerSpace');
+			alert(round + " Has Won");
+			resetSpaces();
+			return acrossTop;
+		}	
+		else{
+			return null;
+		}
+	}
+};
+
+var acrossMid= function($box4, $box5, $box6){
+	var space4 = $('#box4').attr("class");
+	var space5 = $('#box5').attr("class");
+	var space6 = $('#box6').attr("class");
+
+	if ((space4 === space5) && (space5 === space6)){
+		if (space4 === 'playedSpaceX'){
+			console.log ('X has won');
+			$('#box4').addClass('winnerSpace');
+			$('#box5').addClass('winnerSpace');
+			$('#box6').addClass('winnerSpace');
+			alert(round + " Has Won");
+			resetSpaces();
+			return acrossMid;
+		}
+		else if (space4 === 'playedSpaceO'){
+			console.log ('O has won');
+			$('#box4').addClass('winnerSpace');
+			$('#box5').addClass('winnerSpace');
+			$('#box6').addClass('winnerSpace');
+			alert(round + " Has Won");
+			resetSpaces();
+			return acrossMid;
+		}
+		else{
+			return null;
+		}
+	}	
+};
+
+var acrossBot= function($box7, $box8, $box9){
```

Comment:
I think all 3 of your `acrossX` functions could have just been one function `across` that you call by specifying which row to check.